app/validators/bookmark_controller_validator.rb

```diff
@@ -0,0 +1,7 @@
+class BookmarkControllerValidator < ActiveModel::EachValidator
+  @@active_record_tables ||= ActiveRecord::Base.connection.tables.map(&:to_s)
+  def validate_each(record, attribute, value)
+    controllers = ["dashboard"] + (@@active_record_tables + Permission.resources.map {|x| x.tableize }).uniq
+    record.errors[attribute] << _("%{value} is not a valid controller") unless controllers.include?(value)
```

Comment:
%value needs interpolating here, so just add `% {:value => value)` after the `_()` call.